### PR TITLE
upgrade node-inspector package to support node 4.0, fix tests

### PR DESCRIPTION
**Pull request by gmlove**

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    grep  = spawn('grep', ['ssh']);
+
+grep.on('close', function (code, signal) {
+  console.log('child process terminated due to receipt of signal ' + signal);
+});
+
+console.log('pid: ', grep.pid);
+
+// send SIGHUP to process
+grep.kill('SIGHUP');
```
Comment (ilanbiala) [inline test/test.js]: newline

**Comment by ilanbiala** [comment]
@gmlove @fiznool can you guys merge your PRs into 1 so that we can resolve issues with Node 4? @ChrisWren please help merge this in because it affects compatibility with Node 4. Also, we should start testing against Node 4 in this PR to see whether it works or not.

**Comment by fiznool** [comment]
@ilanbiala I believe that the most recent version 0.3.0 of this module now supports node v4.

@ChrisWren if you are up for merging in my options fixes from #28 please let me know and I'll submit a new PR.

**Comment by gmlove** [comment]
Sorry, I missed the notification. Now everything seems good. Thank you all!

**Pull request merged**